tests/Backend/Mixed/SharingTest.php

```diff
@@ -1200,6 +1231,274 @@ function (array $column) {
         $this->assertCount(5, $workspaceTableData);
     }
 
+    private function sharedBucketDetailTest($bucketId, $tableId, $project)
+    {
+        $columnId = $tableId . '.transid';
+        $this->prepareTestMetadata($bucketId, $tableId, $columnId);
+
+        $response = $this->_client2->getSharedBucketDetail(
+            $project['id'],
+            $bucketId,
+            [
+                'include' => 'columns'
+            ]
+        );
+
+        $this->assertArrayHasKey('id', $response);
+        $this->assertArrayHasKey('description', $response);
+        $this->assertArrayHasKey('project', $response);
+        $this->assertArrayHasKey('tables', $response);
+        $this->assertArrayHasKey('created', $response);
+        $this->assertArrayHasKey('lastChangeDate', $response);
+        $this->assertArrayHasKey('dataSizeBytes', $response);
+        $this->assertArrayHasKey('rowsCount', $response);
+        $this->assertArrayHasKey('backend', $response);
+
+        $this->assertArrayHasKey('id', $response['project']);
+        $this->assertArrayHasKey('name', $response['project']);
+
+        $this->assertSame($bucketId, $response['id']);
+        $this->assertEquals($response['project']['id'], $project['id']);
+        $this->assertEquals($response['project']['name'], $project['name']);
+
+        $this->assertArrayHasKey('tables', $response);
+
+        $tableColumns = $this->_client->getTable($tableId)['columns'];
+        foreach ($response['tables'] as $table) {
+            $this->assertArrayNotHasKey('metadata', $table);
+            $this->assertArrayNotHasKey('columnMetadata', $table);
+            $this->assertArrayHasKey('columns', $table);
+            $this->assertCount(5, $table['columns']);
+            foreach ($table['columns'] as $column) {
+                $this->assertTrue(in_array($column, $tableColumns));
+            }
+        }
```

Comment:
Až pocaď by to mohlo využít výsledek toho volání z ř. 1277, ne?

Reply:
https://github.com/keboola/storage-api-php-client/pull/366#discussion_r385650950

Reply:
ok